pytradfri/color.py

```diff
@@ -39,32 +45,32 @@
           for hex, name in COLOR_NAMES.items()}
 
 
-def can_kelvin_to_xy(k):
-    return MIN_KELVIN <= k <= MAX_KELVIN
-
-
-# Only used locally to perform normalization of x, y values
-# Scaling to 65535 range and rounding
-def normalize_xy(x, y):
-    return (int(x*65535+0.5), int(y*65535+0.5))
+def kelvin_to_xyY(T, white_spectrum_bulb=False):
```

Comment:
Do we still need these conversion methods?

Reply:
No, you are right. I made some tests and it works to just set Mireds. I've removed them.